src/api/api.js

```diff
@@ -54,6 +54,9 @@ class Api {
       }
 
       if (message.message) {
+        if (!document.hasFocus()) {
```

Comment:
can we move this code inside `onMessageListener` function?
so the API will not be connected with UI (`document `)